content/soda-gettingstarted/soda-lab-setup/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: SODA Lab Setup
+description: "Setup development SODA Lab"
+weight: 50
+disableToc: false
+---
+
```

Comment:
Consider adding a overall summary of Home lab setup letting the user know what all the sections help with and how it helps with SODA Installation.